From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Mon, 9 May 2022 00:32:04 +0200
Subject: [PATCH] Build: Fix `char*` vs. `const char*` arguments

These arguments are of the wrong constness, which will trip our
compiler.
---
 nebu/include/scripting/nebu_scripting.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/nebu/include/scripting/nebu_scripting.h b/nebu/include/scripting/nebu_scripting.h
index 867eada85698197845f1bf500211f88fba286601..f5d80848390c13aa879cfa60127a5b140deb73d0 100644
--- a/nebu/include/scripting/nebu_scripting.h
+++ b/nebu/include/scripting/nebu_scripting.h
@@ -18,9 +18,9 @@ extern int scripting_GetStringResult(char **s);
 extern int scripting_CopyStringResult(char *s, int len);
 
 
-extern void scripting_RunFile(const char *name);
-extern void scripting_Run(const char *command);
-extern void scripting_RunFormat(const char *format, ...);
+extern void scripting_RunFile(char *name);
+extern void scripting_Run(char *command);
+extern void scripting_RunFormat(char *format, ...);
 extern void scripting_RunGC();
 extern void scripting_Register(const char *name, int(*func) (lua_State *L));
 
